src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -358,6 +358,9 @@
   <data name="SortNotSupported" xml:space="preserve">
     <value>The _sort parameter is not supported.</value>
   </data>
+  <data name="TypeNotPresent" xml:space="preserve">
+    <value>The type is not present.</value>
```

Comment:
Can we be more specific on error message. For example "Bundle type is not present. Possible values are: transaction, batch".